### ISSUE TITLE
trippplets.go

### ISSUE THREAD

**Issue by elrikdante** [Lenguti/challenges#1]
https://github.com/Lenguti/challenges/blob/master/triplets.go

The solution looks good so far, but we need to make sure we can feed it arbitrary
inputs.

Let's add reading from standard in, you may want to just package all this into a little lib that you can import into all your challenges.

**Comment by Lenguti** [comment]
Yes, I like that idea a lot! I will add that to the repo. Thanks for bringing that up.
